pkg/controller/certificaterequests/venafi/venafi.go

```diff
@@ -30,22 +37,22 @@ import (
 	controllerpkg "github.com/jetstack/cert-manager/pkg/controller"
 	"github.com/jetstack/cert-manager/pkg/controller/certificaterequests"
 	crutil "github.com/jetstack/cert-manager/pkg/controller/certificaterequests/util"
-	venafiinternal "github.com/jetstack/cert-manager/pkg/internal/venafi"
-	internalvanafiapi "github.com/jetstack/cert-manager/pkg/internal/venafi/api"
 	issuerpkg "github.com/jetstack/cert-manager/pkg/issuer"
 	logf "github.com/jetstack/cert-manager/pkg/logs"
 )
 
 const (
-	CRControllerName = "certificaterequests-issuer-venafi"
+	CRControllerName         = "certificaterequests-issuer-venafi"
+	VenafiPickupIDAnnotation = "venafi.cert-manager.io/pickupid"
```

Comment:
nit: should be `pickup-id`

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions

> Key names should be all lowercase, with words separated by dashes instead of camelCase
> For instance, prefer foo.kubernetes.io/foo-bar over foo.kubernetes.io/fooBar, prefer desired-replicas over DesiredReplicas